doc/examples/reconst_cti.py

```diff
@@ -0,0 +1,152 @@
+from dipy.io.image import load_nifti
+"""
+CTI analysis requires specific gradient information from the data, often acquired from different gradient tables. In this example, we manually load b-values and b-vectors from two separate sets of files, 'bvals1.bval' and 'bvec1.bvec', and 'temp_data/bvals2.bval' and 'temp_data/bvec2.bvec'. 
+The users should ensure that the data is formatted correctly for the CTI analysis they are performing.
+""" 
+data, affine = load_nifti('temp_data/RB_invivo_cti_data_f3.nii')
+bvals1, bvecs1 = read_bvals_bvecs('temp_data/bvals1.bval',
+                                  'temp_data/bvec1.bvec')
+bvals2, bvecs2 = read_bvals_bvecs('temp_data/bvals2.bval',
+                                  'temp_data/bvec2.bvec')
+                                  """ 
+In this example, the function `load_nifti` is used to load the CTI data from the file 'RB_invivo_cti_data_f3.nii' and returns the data as a nibabel Nifti1Image object along with the affine transformation. The b-values and b-vectors for two different gradient tables are loaded from 'bvals1.bval' and 'bvec1.bvec', and 'bvals2.bval' and 'bvec2.bvec' respectively using the `read_bvals_bvecs` function. These are then  further converted into gradient tables `gtab1` and `gtab2` which are essential for the CTI analysis being conducted.
+"""
+gtab1 = gradient_table(bvals1, bvecs1)
+gtab2 = gradient_table(bvals2, bvecs2)
+"""
+Before fitting the data, we perform some data pre-processing. We first compute a brain mask to avoid unnecessary calculations on the background of the image.
+"""
+maskdata, mask = median_otsu(data, vol_idx=[0, 1], median_radius=4, numpass=2,
+                             autocrop=False, dilate=1)
+"""
+Now that we have loaded and created a mask for the data we can go forward with CTI fitting. For this, the CTI model is first defined for the data’s GradientTable object by instantiating the CorrelationTensorModel object in the following way:
+"""
+ctimodel = cti.CorrelationTensorModel(gtab1, gtab2)
+"""
+To fit the data using the defined model object, we call the fit function of this object.
+"""
+ctifit = ctimodel.fit(data, mask = mask)
+"""
+The fit method for the CTI model produces a CorrelationTensorFit object, which contains the attributes of both the DKI and DTI models. Given that CTI is a layered model built upon the DKI, which itself extends the DTI model, the CorrelationTensorFit instance captures a comprehensive set of parameters and attributes from these underlying models.
+
+For instance, since the DKI model inherently estimates the diffusion tensor parameters, all standard DTI statistics, such as fractional anisotropy (FA), mean diffusivity (MD), axial diffusivity (AD), and radial diffusivity (RD), can be directly extracted from the CorrelationTensorFit instance. Additionally, the CTI offers further insights by evaluating correlation tensors, offering a richer perspective on the tissue's microstructural environment.
```

Comment:
Write instead. For instance, since the CTI model inherently estimates all DTI and DKI statistics, such as the mean, axial and radial diffusivities (MD, AD, RD) and the mean, axial and radial kurtosis (MK, RK, AK).

Add some lines of code retrieving the main DTI/DKI parameters to illustrate what you are saying.